README.md

```diff
@@ -2,7 +2,7 @@
 
 Inspect and manage `node_modules` trees.
 
-![a tree with the word ARBORIST superimposed on it](https://raw.githubusercontent.com/npm/arborist/master/logo.svg?sanitize=true)
+![a tree with the word ARBORIST superimposed on it](https://raw.githubusercontent.com/npm/arborist/main/logo.svg?sanitize=true)
```

Comment:
```suggestion
![a tree with the word ARBORIST superimposed on it](https://raw.githubusercontent.com/npm/arborist/HEAD/logo.svg?sanitize=true)
```
this will work regardless of what the default branch is.